R/inner_combine_linter.R

```diff
@@ -6,6 +6,19 @@
 #'   preferred so that the most expensive part of the operation ([as.Date()])
 #'   is applied only once.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "c(log10(x), log10(y), log10(z))",
+#'   linters = inner_combine_linter()
+#' )
+#'
+#' # okay
```

Comment:
suggestion: an `okay` example like `c(as.Date(x, "%Y/%m/%d"), as.Date(z, "%Y%m%d"))`

Reply:
or keep the theme here and use `log()` with different `base=` arguments

Reply:
Excellent suggestion!

I have used an example with `log(base = )`.